simple/stats/db.py

```diff
@@ -38,16 +44,32 @@
 );
 """
 
+_DELETE_OBSERVATIONS_STATEMENT = "delete from observations"
 _INSERT_OBSERVATIONS_STATEMENT = "insert into observations values(?, ?, ?, ?, ?)"
 
+_INIT_STATEMENTS = [
+    _CREATE_TRIPLES_TABLE,
+    _CREATE_OBSERVATIONS_TABLE,
+    # Clearing tables for now.
+    _DELETE_TRIPLES_STATEMENT,
+    _DELETE_OBSERVATIONS_STATEMENT
+]
+
 
 class Db:
   """Class to insert triples and observations into a sqlite DB."""
 
   def __init__(self, db_file_path: str) -> None:
-    self.db = sqlite3.connect(db_file_path)
-    self.db.execute(_CREATE_TRIPLES_TABLE)
-    self.db.execute(_CREATE_OBSERVATIONS_TABLE)
+    self.db_file_path = db_file_path
+    # If file path is a GCS path, we create the DB in a local temp file
+    # and upload to GCS on commit.
```

Comment:
wonder why need to upload the db to GCS? For large db, this can take time and block?

Reply:
Disabled upload until we have cloud SQL support.